packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -0,0 +1,199 @@
+import { Component, EventEmitter, Inject, Input, OnChanges, OnInit, Output, SimpleChanges } from "@angular/core";
+import { TranslateService } from "@ngx-translate/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+/**
+ * Component to display dropdown list based on the options passed as parameters. The dropdown component is based
+ * on the Angular Material MatSelect.
+ */
+@Component({
+	selector: "stark-dropdown",
+	templateUrl: "./dropdown.component.html"
+})
+export class StarkDropdownComponent implements OnInit, OnChanges {
+
+	/**
+	 * (optional) if the dropdown will contain a default blank option
+	 */
+	@Input()
+	public defaultBlank?: boolean;
+	
+	/**
+	 * HTML "id" attribute of the element.
+	 */
+	@Input()
+	public dropdownId: string;
+	
+	/**
+	 * HTML "name" attribute of the element.
+	 */
+	@Input()
+	public dropdownName: string;
+	
+	/**
+	 * header Text to be displayed as the dropdown's header. Such header will be shown at the top of the
+	 * dropdown options list(dynamically translated via the $translate service if the provided text is defined in the translation keys).
+	 */
+	// TODO reenable once a solution for the replacement of md-select-header as been found
+	// @Input()
+	// public header?: string;
+	/**
+	 * (optional) Whether the dropdown is disabled
+	 */
+	@Input()
+	public isDisabled?: boolean = false;
+	
+	/**
+	 * Text to be displayed as the dropdown's label (dynamically translated via the $translate service if
+	 * the provided text is defined in the translation keys).
+	 */
+	@Input()
+	public label?: string;
+	
+	/**
+	 * (optional) Allows multiple option selection. Setting the attribute to "true" or empty
+	 * will enable this feature.
+	 */
+	@Input()
+	public multipleSelect?: string;
+	
+	/**
+	 * Array of options to be included in the dropdown list. This parameter is a one-way
+	 * binding (one-directional).
+	 */
+	@Input()
+	public options: any[];
+	
+	/**
+	 * (optional) Property name to be used as id of the options defined. If this parameter as
+	 * well as the optionLabelProperty are omitted, then the options parameter is considered to be an array of simple types.
+	 */
+	@Input()
+	public optionIdProperty?: string;
+	
+	/**
+	 * (optional) Property name to be used as label of the options defined (dynamically translated via
```

Comment:
In other modules (stark-core), the `(optional)` is at the end of the description. Could you please do the same here ?